dns/dyndns/src/etc/inc/plugins.inc.d/dyndns.inc

```diff
@@ -132,6 +132,7 @@ function dyndns_list()
         'selfhost' => 'SelfHost',
         'strato' => 'STRATO',
         'zoneedit' => 'ZoneEdit',
+        'all-inkl' => 'All-Inkl.com',
```

Comment:
Can you merge this into #693 for a clean PR?

Also, the list is sorted alphabetically so it should be entry number 2. :)